### ISSUE TITLE
i18ndude filter, seems to be broken under Python3.7

### ISSUE THREAD

**Issue by MrTango** [collective/i18ndude#75]
```
$ ./bin/i18ndude filter src/Products/EasyNewsletter/locales/plone.pot src/Products/EasyNewsletter/locales/EasyNewsletter.pot
Traceback (most recent call last):
  File "./bin/i18ndude", line 16, in <module>
    sys.exit(i18ndude.script.main())
  File "/home/maik/develop/plone/.buildout/eggs/i18ndude-5.3.1-py3.7.egg/i18ndude/script.py", line 742, in main
    errors = arguments.func(arguments)
  File "/home/maik/develop/plone/.buildout/eggs/i18ndude-5.3.1-py3.7.egg/i18ndude/script.py", line 505, in filter
    for msgid in f1_ctl.keys():
RuntimeError: OrderedDict mutated during iteration
```

**Comment by mauritsvanrees** [comment]
Confirmed. So it goes wrong in [these three lines](https://github.com/collective/i18ndude/blob/5.3.1/src/i18ndude/script.py#L505-L507). Let me see if I can quickly fix that.

**Comment by mauritsvanrees** [comment]
Fixed in i18ndude 5.3.2.
